.storybook/webpack.config.ts

```diff
@@ -115,8 +115,9 @@ function useMockedModules(moduleNames) {
 }
 
 // Export a function. Accept the base config as the only param.
-module.exports = async ({ config, mode }) => {
-  const isDevMode = mode === 'development'
+export default async ({ config, mode }) => {
+  const isDevMode = mode.toLowerCase() === 'development'
```

Comment:
It's funny that the mode seems to be `development`, but the entry in `argv` is `dev`.

Reply:
So the entry in `argv` is `dev` because the command which is executed is `{node} path/to/storybook.js {build|dev}`. The mode there is the separate `Webpack` mode, (I think it's coming from the `NODE_ENV` environment variable) which is either `DEVELOPMENT` or `PRODUCTION` (it's changed based on whether Storybook is doing a static build or a dev build).

`config.mode` is `development` | `production`, so  maybe we should be using that instead - this was just fixing a bug I noticed that `isDevMode` was always false.